### PR TITLE
Add hamburger menu

### PR DESCRIPTION
**Pull request by purvitrivedi**

### CODE REVIEW

File: styles/navbar.module.css [hunk 1]
```diff
@@ -82,4 +82,84 @@
...
+        width: 24px;
+    }
+
+    .hamburger {
+        background: none;
+        border: none;
+        display: block;
+    }
+
+    .navbar_content_mobile {
```
Comment (preetjdp) [inline styles/navbar.module.css:106]: So there's a bug with z-index here (check image)

image

One solution would be to add a z-index of 100

```
z-index: 100;
```

**Review by preetjdp — CHANGES_REQUESTED** [review]

**Comment by preetjdp** [comment]
The three-lines and the cross button change places when toggled, this looks a bit weird to me, 

IMO they should be in the same place

what do you think?


https://user-images.githubusercontent.com/27439197/116775719-1f263480-aa82-11eb-8711-2f396a81e1c1.mov

**Comment by purvitrivedi** [comment]
> The three-lines and the cross button change places when toggled, this looks a bit weird to me,
> 
> IMO they should be in the same place
> 
> what do you think?
> 
>  Screen.Recording.2021-05-01.at.1.33.43.PM.mov

Just pushed a fix - let me know what you think! :)

**Review by preetjdp — APPROVED** [review]
LGTM

**Pull request merged**